tests/scripts/thread-cert/v1_2_LowPower_7_1_02_SingleProbeLinkMetricsWithoutEnhancedAck.py

```diff
@@ -0,0 +1,380 @@
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+import unittest
+
+from config import ADDRESS_TYPE
+from mle import LinkMetricsSubTlvType
+from pktverify import consts
+from pktverify.null_field import nullField
+from pktverify.packet_verifier import PacketVerifier
+
+import config
+import thread_cert
+
+LEADER = 1
+SED_1 = 2
+SSED_1 = 3
+
+SERIES_ID = 1
+SERIES_ID_2 = 2
+POLL_PERIOD = 2000  # 2s
+
+
+class LowPower_7_1_02_SIngleProbeLinkMetricsWithoutEnhancedAck(thread_cert.TestCase):
+    USE_MESSAGE_FACTORY = False
+    TOPOLOGY = {
+        LEADER: {
+            'version': '1.2',
+            'name': 'LEADER',
+            'mode': 'rdn',
+            'allowlist': [SED_1, SSED_1],
+        },
+        SED_1: {
+            'version': '1.2',
+            'name': 'SED_1',
+            'mode': '-',
+            'allowlist': [LEADER],
+        },
+        SSED_1: {
+            'version': '1.2',
+            'name': 'SSED_1',
+            'mode': '-',
+            'allowlist': [LEADER],
+        }
+    }
+    """All nodes are created with default configurations"""
+
+    def test(self):
+        self.nodes[LEADER].start()
+        self.simulator.go(config.LEADER_STARTUP_DELAY)
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+
+        self.nodes[SED_1].set_pollperiod(POLL_PERIOD)
+        self.nodes[SED_1].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[SED_1].get_state(), 'child')
+
+        self.nodes[SSED_1].set_csl_period(consts.CSL_DEFAULT_PERIOD)
+        self.nodes[SSED_1].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[SSED_1].get_state(), 'child')
+
+        leader_addr = self.nodes[LEADER].get_ip6_address(ADDRESS_TYPE.LINK_LOCAL)
+
+        # Step 3 - Verify connectivity by instructing each device to send an ICMPv6 Echo Request to the DUT
+        self.assertTrue(self.nodes[SED_1].ping(leader_addr, timeout=POLL_PERIOD / 1000))
+        self.assertTrue(self.nodes[SSED_1].ping(leader_addr))
+        self.simulator.go(5)
+
+        # Step 4 - SED_1 sends a Single Probe Link Metric for RSSI using MLE Data Request
+        # MLE Data Request Payload:
+        # - TLV Request TLV (Link Metrics Report TLV specified)
+        # - Link Metrics Query TLV
+        # -- Link Metrics Query ID Sub-TLV
+        # --- Query ID = 0 (Single Probe Query)
+        # -- Link Metrics Query Options Sub-TLV
+        # --- Metric Type ID Flags
+        # ---- Type / Average Enum = 1  (Exponential Moving Avg)
+        # ---- Metric Enum = 3  (RSSI)
+        # TODO: In this step, SED_1 should set its TxPower to 'High'. But TxPower is not supported on simulation for now.
```

Comment:
`macfilter` command supports changing simulated RSSI value. Can we use that here?